src/pages/platform/login.tsx

```diff
@@ -0,0 +1,79 @@
+import { useEffect, useRef } from 'react'
+import { v4 } from 'uuid'
+
+const jwt = require('jsonwebtoken')
+import { GetServerSideProps } from 'next'
+
+export const getServerSideProps: GetServerSideProps = async (context) => {
+  // TODO: verify token
+
+  const message = {
+    iss: process.env.PROVIDER_URL,
+    aud: ['editor'],
+    // TODO: no idea where this should be coming from
+    sub: '0ae836b9-7fc9-4060-006f-27b2066ac545',
+
+    iat: Date.now(),
+    nonce: v4(),
+
+    // TODO: no idea where this should be coming from
+    'https://purl.imsglobal.org/spec/lti/claim/deployment_id':
+      '8c49a5fa-f955-405e-865f-3d7e959e809f',
+    'https://purl.imsglobal.org/spec/lti/claim/message_type':
+      'LtiDeepLinkingRequest',
+    'https://purl.imsglobal.org/spec/lti/claim/version': '1.3.0',
+    'https://purl.imsglobal.org/spec/lti/claim/roles': [],
+
+    // 'https://purl.imsglobal.org/spec/lti/claim/launch_presentation': {
+    //   document_target: 'iframe',
+    //   height: 320,
+    //   width: 240,
+    // },
+
+    'https://purl.imsglobal.org/spec/lti-dl/claim/deep_linking_settings': {
+      // TODO: host
+      deep_link_return_url: 'http://localhost:3000/platform/done',
+      accept_types: ['ltiResourceLink'],
+      // accept_media_types: 'image/*,text/html',
+      // accept_presentation_document_targets: ['iframe', 'window', 'embed'],
```

Comment:
`accept_presentation_document_targets` it is actually required.
(see https://www.imsglobal.org/spec/lti-dl/v2p0#deep-linking-request-message, search for  3.4.1 Deep linking settings)